src/test/java/org/fundacionjala/pivotal/hooks/Hooks.java

```diff
@@ -1,20 +1,56 @@
 package org.fundacionjala.pivotal.hooks;
 
+import com.jayway.restassured.response.Response;
 import cucumber.api.java.After;
-import org.fundacionjala.pivotal.core.api.APICommons;
+
 import org.fundacionjala.pivotal.core.api.APIRequestManager;
+import org.fundacionjala.pivotal.core.api.APICommons;
+import org.fundacionjala.pivotal.core.api.APIResponseName;
+
+import java.util.Map;
 
 /**
  * Hooks.java
  * Class that will have hooks for scenarios.
  */
 public class Hooks {
 
+    private final APIResponseName date;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "date" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)